README.md

```diff
@@ -433,60 +440,67 @@ issuing stock is how companies often choose to do this.
-    having a separate certificate for it.
-- 🄳 [**Outstanding shares**](http://www.investopedia.com/terms/o/outstandingshares.asp) refer to
-  the total number of shares held by all shareholders.
-  This number starts at an essentially arbitrary value (such as 10 million) when the company
-  is created, and thereafter will increase as new shares are added (issued) and granted to
-  people in exchange for money or services.
-  Outstanding shares may increase or decrease for other reasons too (such as stock splits
-  and share buybacks, which we won’t get into here).
-  Later, we discuss [several subtleties](#counting-shares) in how shares are counted.
-  - 🚧 What is a good overview on stock splits and share buyback.
-    Key resources?
-- 🄳 Any shareholder has a **percentage ownership** in the company, determined by dividing the
-  number of shares they own by the number of outstanding shares.
-  Although stock paperwork will always list numbers of shares, if share value is uncertain,
-  percentage ownership is often a more meaningful number, particularly if you know or can
-  estimate a likely valuation of the company.
-  Even if the number of shares a person has is fixed, their percentage ownership will change
-  over time as the outstanding shares change.
-  Typically, this number is presented in percent or
-  [**basis points**](https://www.investopedia.com/terms/b/basispoint.asp) (hundredths of a percent).
+🄳 [**Stock**](https://en.wikipedia.org/wiki/Stock) is a legal invention that represents
+ownership in a company. [**Shares**](https://en.wikipedia.org/wiki/Share_(finance)) are portions
+of stock that allow a company to grant ownership to a variety of people or other companies
+in flexible ways.
+Each **shareholder (or stockholder)**, as these owners are called, holds a specific number
+of shares. Founders, investors, employees, board members, contractors, advisors, and other
+companies, like law firms, can all be shareholders.
+
+🄳 Stock ownership is often formalized on **stock certificates**, which are fancy pieces of
+paper that prove who owns the stock.
+
+- Sometimes you have stock but don’t have the physical certificate, as it may be held for
+  you at a law office.
+  Some companies now manage their ownership through online services called *ownership
+  management platforms*, such as [Carta](https://carta.com/). If the company you work for uses
+  an ownership management platform, you will be able to view your stock certificates and
+  stock values online.
+  Younger companies may also choose to keep their stock *uncertificated*, which means your
+  sole evidence of ownership is your contracts with the company, and your spot on the
+  company’s [capitalization table](https://en.wikipedia.org/wiki/Capitalization_table), without
+  having a separate certificate for it.
+
+🄳 [**Outstanding shares**](http://www.investopedia.com/terms/o/outstandingshares.asp) refer to
```

Comment:
Outstanding shares and percentage ownership still feel right to me in "Counting shares," or perhaps those sections should be combined. Confusing why they're separated by companies and startups info...